es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log(`LearningFuze: ${address}`);
```

Comment:
Do not change anything which is not concatenation:

---

![image](https://user-images.githubusercontent.com/16143771/174704822-1ee2936e-2380-41d5-b112-8a7edb54fa2e.png)

---

![image](https://user-images.githubusercontent.com/16143771/174704783-c0bc16dc-ae63-42f0-a752-55a861cab4a7.png)

Reply:
Is there a separate term for this? I thought it was just a different way of concatenating.

Reply:
@JoeDeiRossi Passing the `console.log` method two arguments is different from a single concatenated argument.